lib/FFI/Platypus/Type/Enum.pm

```diff
@@ -202,9 +202,21 @@ sub ffi_custom_type_api_1
   foreach my $value (@values)
   {
     my $name;
+    my @aliases;
+
     if(is_plain_arrayref $value)
     {
-      ($name,$index) = @$value;
+      my %opt;
+      if(@$value % 2)
```

Comment:
This is a bit of a "clever" (in the bad sense of the word) way of getting around having to pass in `undef`, but I think it is the lesser evil here.